### PR TITLE
Rework training links pages into two resources pages

### PR DESCRIPTION
**Pull request by mtbc**
## Description
Following discussion in #14, I've tried:
* separating the two link lists into separate pages
* making their links open in new tabs
* called the header "Resources" instead of "Training".

Could be that I should add a redirect though, if so, then to which?

## Motivation and Context
Attempts to sufficiently advance #14.

## Checklist:

- [ ] I have posted the link for the PR in the usrse slack (#website) to ask for reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc @usrse-maintainers

### CODE REVIEW

**Comment by cosden** [comment]
Definitely the right call on changing the high level menu to resources!

Right now, the separation into organization and RSE resources seems unnecessary, but I can see the need in the future as the lists grow. I'd like to suggest changing the order and renaming, so it goes "RSE Resources" then "Related Organizations."

On a related note, the Education and Training working group is preparing (and plans to maintain) a page of links to training materials.  It's not ready yet, but where would that go? As a third menu item? Or would you think it goes in resources under a new header?

**Comment by vsoch** [comment]
This looks great! We are going to need to have redirects from old pages to here, e.g.,

-  /links/ -> /resources/

And I think training went to /events-training/ which should redirect to events? But that's out of scope for this PR.

**Comment by vsoch** [comment]
Redirect instructions -> https://github.com/vsoch/usrse.github.io#5-how-do-i-add-a-page-redirect (or just find an existing one and copy the example).

**Comment by mtbc** [comment]
> As a third menu item?

That's my thinking, yes. We have the popup submenu framework available, let's put it to use.

**Comment by mtbc** [comment]
> Could be that I should add a redirect though, if so, then to which?

Let's make it to "RSE Resources" then.

**Comment by mtbc** [comment]
> /events-training/ which should redirect to events

It looks as if `pages/redirects/events.md` is doing that for us.

**Comment by vsoch** [comment]
I must have added it! Then we just need /links/ -> /resources/.

File: pages/resources/rses.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+_These resources can be helpful to establish RSE positions, groups, or further understanding of research software engineering._
+
+ - [How to start an RSE Group](https://www.software.ac.uk/blog/2019-08-19-how-do-you-start-rse-group){:target="_blank"} as a leader
+ - [Community Documents](https://github.com/USRSE/usrse.github.io/wiki){:target="_blank"}
+ - [RSE Starter Pack](http://rseng.github.io/starter-pack/#/){:target="_blank"} getting started to create an RSE community as an RSE
+ - [The Story of the Research Software Engineer](https://www.youtube.com/watch?v=trAfA9VWLTQ){:target="_blank"} video introduction to Research Software Engineering
+ - [US-RSE Logos](https://github.com/usrse/logo){:target="_blank"} repository
+ - [US-RSE Slack](https://usrse.slack.com/){:target="_blank"}
```
Comment (cosden) [inline pages/resources/rses.md:15]:
```suggestion
 - [US-RSE Slack](https://usrse.slack.com/){:target="_blank"}
 
 _See the [Related Organizations resource page]({{ site.baseurl }}/resources/organizations/ ) for links to additional useful resources by other organizations related to Research Software Engineering._
```

Comment (cosden) [reply pages/resources/rses.md:15]: I think a link (with a short description) to the other resource pages makes sense so people don't stop or forget.   This is just a suggestion, there might be better ways to say it.

Comment (vsoch) [reply pages/resources/rses.md:15]: As soon as this point is resolved (commit or decided not to) I'm good to merge

**Comment by mtbc** [comment]
https://ideas-productivity.org/ seems to be down but don't want to remove the links until it's less likely to be just a transient issue.

**Review by vsoch — APPROVED** [review]
I'm okay merging while the URL is dead, it will come back :)

**Pull request merged**